niri-config/src/lib.rs

```diff
@@ -2370,6 +2370,65 @@ pub enum PreviewRender {
     ScreenCapture,
 }
 
+#[derive(Debug, Clone)]
+pub enum ConfigPath {
+    /// Load the config only from this path, never create it.
+    Explicit(PathBuf),
+    /// Normal operation: Prioritize the user path, fallback to the system path,
+    /// else maybe create the user path if we're starting (but not reloading).
```

Comment:
First line of doc comment should be a brief title. So please adjust, or change to `//` (even tho we currently don't generate rustdocs for this)